bl_config.h

```diff
@@ -0,0 +1,13 @@
+#ifndef BL_CONFIG_H_INCLUDED
+#define BL_CONFIG_H_INCLUDED
+
+#include "bl_version.h"
+
+#define BL_WD_TO_MAX_MS         8000
+
+#define STR_INDIR(x) #x
```

Comment:
Please create a new header file called bl_utils.h and move those two macros to it.
Also add a comment there explaining the use case of them